mitmproxy/proxy/layers/quic.py

```diff
@@ -1121,6 +1143,7 @@ class ClientQuicLayer(QuicLayer):
 
     server_tls_available: bool
     """Indicates whether the parent layer is a ServerQuicLayer."""
+    msgs: list[bytes]
```

Comment:
```suggestion
    handshake_datagram_buf: list[bytes]
```

(or anything else that's more descriptive than `msgs`. :)